src/upendra/algo/stack/Queue_implement_using_Stack.java

```diff
@@ -0,0 +1,88 @@
+package upendra.algo.stack;
+import upendra.algo.queue.*;
+import upendra.algo.stack.*;
+public class Queue_implement_using_Stack implements Queue,Stack{
```

Comment:
class name should be in cameras, we should follow java naming convention.
see the links https://www.javatpoint.com/java-naming-conventions

Reply:
done